packages/react-dom/src/__tests__/ReactDOMFizzSuppressHydrationWarning-test.js

```diff
@@ -702,4 +716,173 @@ describe('ReactDOMFizzServerHydrationWarning', () => {
       </div>,
     );
   });
+
+  // @gate experimental && enableClientRenderFallbackOnTextMismatch && enableClientRenderFallbackOnHydrationMismatch
+  it('#24384: Suspending should halt hydration warnings while still allowing siblings to warm up', async () => {
```

Comment:
This test seems exactly the same as the one in ReactDOMFizzServer-test. Why is that?

Also, I intended this test file primarily for testing the behavior of `suppressHydrationWarning`. So unless we're testing that feature (should we, btw?), they probably belong in the other file.

Reply:
I was not, I missed that the file was functional not conceptual and drew the wrong association. The other odd thing was I did remove it from the other file but it still showed up in the diff. I wonder if my force pushing confused github. either way it's fixed now that I moved the tests back :)